core/backend-itwin-client/src/oidc/introspection/IntrospectionResponseCache.ts

```diff
@@ -8,52 +8,60 @@
 
 import { IntrospectionResponse } from "./IntrospectionResponse";
 
-/**
+/** The IntrospectionResponseCache is a simple dictionary of [[IntrospectionResponse]] that are currently in use.
+ * The entries will removed automatically if their tokens expire.
  * @alpha
- * The class UserTokenCache is a simple dictionary of tokens that are currently in use. The entries
- * will removed automatically if their tokens expire.
  */
 export abstract class IntrospectionResponseCache {
-  /**
-   * Adds the given token to the cache. The token will removed automatically from the cache if
-   * it's about to expire.
-   * @param key       Key of the token entry.
-   * @param token     UserToken that is associated with the key.
+  /** Adds the given response to the cache. The response will be added if it has not yet expired.
+   * @param key       A unique string to identify the response within the cache.
+   * @param response  A response associated with the key.
    */
   public async add(key: string, response: IntrospectionResponse): Promise<void> {

```

Comment:
Seems we need a way to cleanup stale entries irrespective of the access token - otherwise the cache will keep growing over time as the access token changes. 

Since the issue is unrelated to the PR, I'm just adding this note.